docs/reference/src/main/asciidoc/ri-spi.asciidoc

```diff
@@ -903,237 +900,103 @@ ifdef::generate-index-link[link:configure.html#external_config[Defining external
 
 === Migration notes
 
-This part of the appendix documents the changes in Weld across major and
-minor releases that an integrator should be aware of. These changes
-mostly touch changes in the SPI or in the container contract.
-
-==== Migration from Weld 1.x to 2.0
```

Comment:
I've deleted all migration notes and added a new section for Weld 4 -> Weld 5 because I thought the section is getting messy and you can always browse older documentation for information on older migrations.
If you disagree, we can keep this (or keep last two versions perhaps?).

Reply:
+1